src/code/z_scene.c

```diff
@@ -231,8 +231,8 @@ void Scene_CommandSpecialFiles(PlayState* play, SceneCmd* cmd) {
         gSegments[5] = VIRTUAL_TO_PHYSICAL(play->objectCtx.status[play->objectCtx.subKeepIndex].segment);
     }
 
-    if (cmd->specialFiles.cUpElfMsgNum != 0) {
-        play->cUpElfMsgs = Play_LoadFile(play, &sNaviMsgFiles[cmd->specialFiles.cUpElfMsgNum - 1]);
+    if (cmd->specialFiles.naviHintFileId != NAVI_HINTS_NONE) {
+        play->naviHints = Play_LoadFile(play, &sNaviHintFiles[cmd->specialFiles.naviHintFileId - 1]);
```

Comment:
It seems more consistent/sensible to use "hints" as plural basically everywhere?
```suggestion
    if (cmd->specialFiles.naviHintsFileId != NAVI_HINTS_NONE) {
        play->naviHints = Play_LoadFile(play, &sNaviHintsFiles[cmd->specialFiles.naviHintsFileId - 1]);
```

this may apply to the z64hint_commands.h filename too, not sure

Reply:
sticking a plural in the middle there just doesnt really sound right to me. "the hint file is a file of hints" seems like the natural phrasing to me

Reply:
"hint file" vs "file with hints", alright

btw with naming the system "Hints" why is all this using "navi hints"

Reply:
this stuff only pertains to navis hints, ie those accessed with `Hints_GetNaviTextId`. sarias hints are seperate. I may not understand your question though

Reply:
You understood correctly, I'm stupid, nevermind